Validation feature fails over missing default constructor

I put the following in my pom.xml

```
      <plugin>
        <!-- Description: https://github.com/ktoso/maven-git-commit-id-plugin -->
        <groupId>pl.project13.maven</groupId>
        <artifactId>git-commit-id-plugin</artifactId>
        <version>2.2.2</version>
        <executions>
          <execution>
            <id>get-the-git-infos</id>
            <phase>validate</phase>
            <goals>
              <goal>revision</goal>
              <goal>validateRevision</goal>
            </goals>
          </execution>
        </executions>
        <configuration>
          <dateFormat>yyyy-MM-dd '@' HH:mm:ss z</dateFormat>
          <validationProperties>
            <!-- verify that the current repository is not dirty -->
            <validationProperty>
              <name>validating git dirty</name>
              <value>${git.dirty}</value>
              <shouldMatchTo>false</shouldMatchTo>
            </validationProperty>
          </validationProperties>
        </configuration>
      </plugin>
```

When I build I get this error:

> [ERROR] Failed to execute goal pl.project13.maven:git-commit-id-plugin:2.2.2:validateRevision (get-the-git-infos) on project yauaa-parent: Unable to parse configuration of mojo pl.project13.maven:git-commit-id-plugin:2.2.2:validateRevision for parameter validationProperty:  Cannot create instance of class pl.project13.maven.validation.ValidationProperty: pl.project13.maven.validation.ValidationProperty.<init>() -> [Help 1]

The underlying error is:

> Caused by: java.lang.NoSuchMethodException: pl.project13.maven.validation.ValidationProperty.<init>()
> 	at java.lang.Class.getConstructor0(Class.java:3082)
> 	at java.lang.Class.newInstance(Class.java:412)

The fix in my situation was really simple: Add a default constructor. 
I'll put up a pull request.

--- a/src/main/java/pl/project13/maven/validation/ValidationProperty.java
+++ b/src/main/java/pl/project13/maven/validation/ValidationProperty.java
@@ -23,6 +23,8 @@ public class ValidationProperty {
   private String value;
   private String shouldMatchTo;
 
+  public ValidationProperty(){};
+
   ValidationProperty(String name, String value, String shouldMatchTo) {
     this.name = name;
     this.value = value;

